src/libraries/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -7684,6 +7683,23 @@ public static string NormalizeToJsonNetFormat(this string json)
                         sb.Append("u002f");
                     }
                 }
+                // Convert > to \u003e
+                else if (json[i] == '>')
+                {
+                    i++;
+                    sb.Append("\\u003e");
+                }
+                // Convert < to \u003c
+                else if (json[i] == '<')
+                {
+                    i++;
+                    sb.Append("\\u003c");
+                }
+                // Remove .0
+                else if (json[i] == '.' && json[i+1] == '0')
```

Comment:
This will also replace `.0` anywhere else in the payload (e.g: a property name). 
This currently doesn't fail since we don't have any case that contains `.0` but it could be problematic in the future.